src/pages/conversation/ConversationPage.tsx

```diff
@@ -170,23 +175,27 @@ const ConversationPage = () => {
     };
   }, [ydoc, editor, selectedCommitFile]);
 
-  const handleEditorMount = (editorInstance: editor.IStandaloneCodeEditor) => {
+  const onEditorMount = (editorInstance: editor.IStandaloneCodeEditor) => {
     setEditor(editorInstance);
   };
 
-  const handleDiffEditorMount = (diffEditor: editor.IStandaloneDiffEditor) => {
+  const onDiffEditorMount = (diffEditor: editor.IStandaloneDiffEditor) => {
     const modifiedEditor = diffEditor.getModifiedEditor();
     setEditor(modifiedEditor);
   };
 
+  const toggleDrawBoard = () => {
+    setDrawBoard((prev) => !prev);
+  };
+
```

Comment:
toggle을 쓰고 싶다면 다음과 같은 코드로 쓸 수 있어요~

```suggestion
const [drawBoard, toggleDrawBoard] = useReducer(false, (state) => !state);
```

Reply:
저렇게쓰면 사이드이펙트가 줄어들겟죠?
그리고 코드가 짧고 오히려 익숙해진다면 알기가 더 편해질꺼에요

Reply:
감사합니다. 이번기회에 reducer에 대해서 공부해보겠습니다.!